dotcom-rendering/src/web/components/App.tsx

```diff
@@ -11,16 +11,6 @@ import { GetMatchNav } from '@frontend/web/components/GetMatchNav';
 import { StickyBottomBanner } from '@root/src/web/components/StickyBottomBanner/StickyBottomBanner';
 import { SignInGateSelector } from '@root/src/web/components/SignInGate/SignInGateSelector';
 
-import {
-	QandaAtom,
-	GuideAtom,
-	ProfileAtom,
-	TimelineAtom,
-	ChartAtom,
-	PersonalityQuizAtom,
-	KnowledgeQuizAtom,
-} from '@guardian/atoms-rendering';
-
 import { AudioAtomWrapper } from '@frontend/web/components/AudioAtomWrapper';
```

Comment:
What's happening to the audio atom? Will this be handled by a separate PR?

Reply:
Good spot. I've added a ticket for this one.